### ISSUE TITLE
Wrong allocation limit

### ISSUE THREAD

**Issue by Leon-JavaScript** [pelican-dev/panel#784]
### Current Behavior

In the server settings tab in the client area, the `Allocation Limit` always shows 1 more than the actual limit.
![image](https://github.com/user-attachments/assets/62657d83-1b23-4e78-be5a-f15c2d4edcce)
![image](https://github.com/user-attachments/assets/c13740ad-bfac-4866-8ebc-6dcc8906d67d)

### Expected Behavior

It should show the correct allocation limit.

### Steps to Reproduce

1. Create a server with at least 1 on the allocation limit.
2. Look at the settings tab in the client area
*It says you can create 1 more allocation than the limit.*

### Panel Version

Tested on 1.0.0-beta14 and canary (157fa45)

### Wings Version

1.0.0-beta8

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by Boy132** [comment]
That's intentional. The allocation is for _additional_ allocations. And every server has at least 1 allocation.
So if you set the allocation limit to 3 you can actually have up to 4 allocation. (the default one + 3 additional)

**Comment by Leon-JavaScript** [comment]
But in both client areas it doesnt allow you to create the 1 allocation over the shown limit.

**Comment by RMartinOscar** [comment]
https://github.com/pelican-dev/panel/blob/8cc91b0747847ef30d1bc08bcf49b6f468bc8630/config/panel.php#L92-L94

Client aren't allowed to create allocations by default you need to check and fill
![image](https://github.com/user-attachments/assets/2b475c9c-7cf6-49fb-9d3f-2ee43dfce522)

**Comment by Leon-JavaScript** [comment]
Even if you have the client allocation creation enabled, it doesn't allow you to set 3 allocations if you have set 2 in the settings of the server. I just tested it in the demo panel.

![image](https://github.com/user-attachments/assets/6fe259d1-5aaa-4cfb-8a9d-7a7322387697)
In the server settings I set the allocations to 2, in the settings it shows 3.

<br>

![image](https://github.com/user-attachments/assets/e0252eb1-39d7-4c75-9eca-3b6c97f2e678)
In the network tab, it shows that the server reached the maximum allocations with 2 allocations.
